### ISSUE TITLE
[BUG] net::ERR_INSUFFICIENT_RESOURCES

### ISSUE THREAD

**Issue by magrinj** [Sparticuz/chromium#17]
## Environment
* `chromium` Version: 108.0.1
* `puppeteer` / `puppeteer-core` Version: 19.2.2
* Node.js Version: 14.x
* Lambda / GCF Runtime: Node.js 14.x

## Expected Behavior

`goTo` should not fail with error

## Current Behavior

`goTo` failed with error `net::ERR_INSUFFICIENT_RESOURCES`

## Steps to Reproduce

Use same instance of puppeteer to scrape 21 pages.

## Possible Solution

Use another binary: https://github.com/puppeteer/puppeteer/issues/2226

**Comment by Sparticuz** [comment]
You can also try increasing the amount of ram available to your instance

**Comment by magrinj** [comment]
@Sparticuz I've steady tried, It's actually set to 10Go and only using 2Go when the error happened

**Comment by benjaminDanis** [comment]
seeing the same error. ram available is not an issue:
`Billed Duration: 7133 ms	Memory Size: 3008 MB	Max Memory Used: 978 MB	Init Duration: 353.94 ms`

**Comment by FreddieFruitSticks** [comment]
Hey so I noticed the above error was often preceded by `"errorMessage": "[Errno 28] No space left on device" ` When I solved this error using the solution here https://github.com/puppeteer/puppeteer/issues/1791 the above error also went away.

I believe it has to do with lambda reuse. went from roughly 20 instances of this error per 200 requests down to 0. Hope this helps.

I am using AWS containerised lambdas with `"puppeteer-core": "^19.2.2"`
